arch/xtensa/src/esp32/esp32_wifi_adapter.c

```diff
@@ -1854,6 +1937,58 @@ static void esp_evt_work_cb(FAR void *arg)
     }
 }
 
+/****************************************************************************
+ * Name: wifi_env_is_chip
+ *
+ * Description:
+ *   Config chip environment
+ *
+ * Returned Value:
+ *   True if on FPGA or false if not.
+ *
+ ****************************************************************************/
+
+static bool wifi_env_is_chip(void)
+{
+  return true;
+}
+
+/****************************************************************************
+ * Name: wifi_set_intr
+ *
+ * Description:
+ *   Do nothing
+ *
+ * Input Parameters:
+ *     cpu_no      - The CPU which the interrupt number belongs.
+ *     intr_source - The interrupt hardware source number.
+ *     intr_num    - The interrupt number CPU.
+ *     intr_prio   - The interrupt priority.
+ *
+ * Returned Value:
+ *     None
+ *
+ ****************************************************************************/
+
+static void wifi_set_intr(int32_t cpu_no, uint32_t intr_source,
+                             uint32_t intr_num, int32_t intr_prio)
+{
+  wlinfo("cpu_no=%d, intr_source=%u, intr_num=%u, intr_prio=%d");
+}
+
+/****************************************************************************
+ * Name: wifi_clear_intr
+ *
+ * Description:
+ *   Don't support
+ *
+ ****************************************************************************/
+
+static void IRAM_ATTR wifi_clear_intr(uint32_t intr_source,
+                                         uint32_t intr_num)
+{
+}
+
```

Comment:
Are these functions to be used later?

Reply:
These functions are members of WiFi adapter structure, so we should add these, although they do nothing. ESP32's some functions are empty in esp-idf, too.